src/tracing/spanProcessor.js

```diff
@@ -1,16 +1,15 @@
 export class SpanProcessor {
   constructor(exporter) {
     this.exporter = exporter;
+    this.pendingSpans = new Map()
   }
 
   onStart(span, _parentContext) {
-    pendingSpans.set(span.spanContext.spanId, span);
+    this.pendingSpans.set(span.span.spanContext.spanId, span);
   }
 
   onEnd(span) {
     this.exporter.export([span.export()])
-    pendingSpans.delete(span.spanContext.spanId);
+    this.pendingSpans.delete(span.span.spanContext.spanId);
```

Comment:
There's a span in my span! What type is the `span` parameter?

Reply:
It's a `Span` instance.

Reply:
Oh, got it. So one is the `Span`, and the other is the opentelemetry key/value pairs? Kind of like the "raw" description of the span.

I don't know if it's a JS thing, but I find it a bit weird that `Span` has a `span` instance var that's not a `Span`. Why not call it `rawSpan` or something more explicit?

Reply:
Won't change or change later.